setup.cfg

```diff
@@ -178,8 +178,11 @@ ignore = E203,E231,E731,W503,W605
 # Settings for Mypy: static type checker for Python 3
 [mypy]
 ignore_missing_imports = True
+# TODO: Change check_untyped_defs to True
```

Comment:
what is this TODO? It's already set to True

Reply:
Oops. I'll remove this.